tests/javascripts/preventDuplicateFormSubmissions.test.js

```diff
@@ -61,7 +61,7 @@ describe('Prevent duplicate form submissions', () => {
 
     helpers.triggerEvent(button, 'click');
 
-    expect(formSubmitSpy.mock.calls.length).toEqual(2);
+    expect(formEventSpy.mock.calls.length).toEqual(0);
```

Comment:
Why is this changing to 0?

Reply:
Instead of testing how many times the form is submitted, it's testing how many times `event.preventDefault()` was called. Since we should be able to submit the form again we expect no calls to `preventDefault`

Reply:
Previously, the code pipeline was `JSDOM.requestSubmit --triggers--> ourCodeOnSubmit --triggers sometimes--> JSDOM.submitSpy` and we could verify that the submit spy was only called once. Now, `JSDOM.requestSubmit` raises a NotImplemented error and if we just mock it out it always is called twice and our code never gets entered to verify that it's working properly.

Reply:
Gotcha, thanks!